voluncheer/chatroom/urls.py

```diff
@@ -2,9 +2,17 @@
 
 from . import views
 
-app_name = 'chatroom'
+app_name = "chatroom"
```

Comment:
Does this actually do anything? I can't find where it is used. Plus it doesn't match Python's style guide for constants which is UPPER_SNAKE_CASE.

Reply:
This is done by Flask8 autocorrect on vs code, so I suppose this is what flask8 told us to follow?